flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -139,6 +140,69 @@ public void shouldSubmitToJobClient() {
 	}
 
 	/**
+	 * Tests that invalid detached mode programs fail.
+	 */
+	@Test
+	public void testDetachedMode() throws Exception{
+		jobManagerSystem.actorOf(Props.create(SuccessReturningActor.class), JobManager.JOB_MANAGER_NAME());
+		Client out = new Client(config);
+
+		try {
+			PackagedProgram prg = new PackagedProgram(TestExecuteTwice.class, "/dev/random", "/tmp");
```

Comment:
I was following the code in `testGetExecutionPlan`. Can you elaborate a bit more?